### PR TITLE
Create service discovery.md

### PR DESCRIPTION
**Pull request by akshayamadhuri**
### Describe your changes

###2058

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/hi/service discovery.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: सेवा खोज (Service Discovery)
```
Comment (aj11anuj) [inline content/hi/service discovery.md:2]:
```suggestion
title: सर्विस डिस्कवरी (Service Discovery)
```

File: content/hi/service discovery.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+status: completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+सेवा की खोज एक सेवा बनाने वाले व्यक्तिगत उदाहरणों को खोजने की प्रक्रिया है। एक सर्विस डिस्कवरी टूल विभिन्न नोड्स या एंडपॉइंट्स का ट्रैक रखता है जो सेवा बनाते हैं।
```
Comment (aj11anuj) [inline content/hi/service discovery.md:9]:
```suggestion
'सर्विस डिस्कवरी' एक सेवा बनाने वाले व्यक्तिगत उदाहरणों को खोजने की प्रक्रिया है। एक सर्विस डिस्कवरी टूल, सर्विस बनाने वाले विभिन नोड्स या एंड-पॉइंट्स को ट्रैक करता है।
```

File: content/hi/service discovery.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## समस्या यह संबोधित करती है
+
+क्लाउड नेटिव आर्किटेक्चर गतिशील और तरल हैं, जिसका अर्थ है कि वे लगातार बदल रहे हैं। एक कंटेनरीकृत ऐप संभवतः अपने जीवनकाल में कई बार शुरू और बंद हो जाएगा। हर बार ऐसा होने पर, इसका एक नया पता होगा और कोई भी ऐप जो इसे खोजना चाहता है उसे नई स्थान जानकारी प्रदान करने के लिए एक टूल की आवश्यकता होगी।
```
Comment (aj11anuj) [inline content/hi/service discovery.md:13]:
```suggestion
क्लाउड नेटिव आर्किटेक्चर गतिशील और तरल हैं, जिसका अर्थ है कि वे लगातार बदलते रहेते हैं। एक कंटेनरीकृत ऐप संभवतः अपने जीवनकाल में कई बार शुरू और बंद होगा। हर बार ऐसा होने पर, इसका एक नया पता होगा और कोई भी ऐप जो इसे खोजना चाहता है उसे नए स्थान की जानकारी प्रदान करने के लिए एक टूल की आवश्यकता होगी।
```

File: content/hi/service discovery.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+## यह कैसे मदद करता है
+
+सर्विस डिस्कवरी नेटवर्क के भीतर ऐप्स का ट्रैक रखती है ताकि जरूरत पड़ने पर वे एक-दूसरे को ढूंढ सकें। यह व्यक्तिगत सेवाओं को खोजने और संभावित रूप से पहचानने के लिए एक सामान्य स्थान प्रदान करता है। सेवा खोज इंजन डेटाबेस-जैसे उपकरण हैं जो इस बारे में जानकारी संग्रहीत करते हैं कि कौन सी सेवाएँ मौजूद हैं और उनका पता कैसे लगाया जाए।
```
Comment (aj11anuj) [inline content/hi/service discovery.md:17]:
```suggestion
सर्विस डिस्कवरी नेटवर्क के भीतर ऐप्स का ट्रैक रखती है ताकि जरूरत पड़ने पर वे एक-दूसरे को ढूंढ सकें। यह व्यक्तिगत सेवाओं को खोजने और संभावित रूप से पहचानने के लिए एक सामान्य स्थान प्रदान करता है। सर्विस की खोज करने वाले सर्च-इंजन डेटाबेस-जैसे उपकरण हैं, जो इस बारे में जानकारी संग्रहीत करते हैं कि कौन सी सेवाएँ मौजूद हैं और उनका पता कैसे लगाया जाए।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

**Review by aj11anuj — APPROVED** [review]
LGTM, 👍

File: content/hi/service discovery.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+title:  सर्विस डिस्कवरी  (Service Discovery)
+status: completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+'सर्विस डिस्कवरी' एक सेवा बनाने वाले व्यक्तिगत उदाहरणों को खोजने की प्रक्रिया है। एक सर्विस डिस्कवरी टूल, सर्विस बनाने वाले विभिन नोड्स या एंड-पॉइंट्स को ट्रैक करता है।
+
+## समस्या यह संबोधित करती है
```
Comment (bishal7679) [inline content/hi/service discovery.md:11]:
```suggestion
## समस्या
```

File: content/hi/service discovery.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+क्लाउड नेटिव आर्किटेक्चर गतिशील और तरल हैं, जिसका अर्थ है कि वे लगातार बदलते रहेते हैं। एक कंटेनरीकृत ऐप संभवतः अपने जीवनकाल में कई बार शुरू और बंद होगा। हर बार ऐसा होने पर, इसका एक नया पता होगा और कोई भी ऐप जो इसे खोजना चाहता है उसे नए स्थान की जानकारी प्रदान करने के लिए एक टूल की आवश्यकता होगी।
+
+## यह कैसे मदद करता है
```
Comment (bishal7679) [inline content/hi/service discovery.md:15]:
```suggestion
## समाधान
```

File: content/hi/service discovery.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+title:  सर्विस डिस्कवरी  (Service Discovery)
+status: completed
+category: अवधारणा
+---
+
+## यह क्या है 
+
+'सर्विस डिस्कवरी' एक सेवा बनाने वाले व्यक्तिगत उदाहरणों को खोजने की प्रक्रिया है। एक सर्विस डिस्कवरी टूल, सर्विस बनाने वाले विभिन नोड्स या एंड-पॉइंट्स को ट्रैक करता है।
```
Comment (bishal7679) [inline content/hi/service discovery.md:9]:
```suggestion
सर्विस डिस्कवरी एक सेवा बनाने वाले व्यक्तिगत उदाहरणों को खोजने की प्रक्रिया है। एक सर्विस डिस्कवरी टूल, सर्विस बनाने वाले विभिन नोड्स या एंड-पॉइंट्स को ट्रैक करता है।
```

**Review by bishal7679 — APPROVED** [review]
LGTM!
Thank you @akshayamadhuri :sparkles:

**Pull request merged**